### PR TITLE
Stage for Main

### PR DESCRIPTION
**Pull request by PitaFLAME**
Launch day overhaul.  Last minute additions have been added.

### CODE REVIEW

File: yarn.lock
Comment (0xfr4nkie) [inline yarn.lock:1]: How did this get here? This is why Vercel is trying to build using Yarn.

File: package.json
Comment (0xfr4nkie) [inline package.json:1]: Recommend specifying with package manager we are using by default. [Source](https://stackoverflow.com/questions/71747609/how-to-specify-packagemanager-in-package-json)

File: public/assets/people/Frank_Lin.jpg
Comment (0xfr4nkie) [inline public/assets/people/Frank_Lin.jpg:1]: There is already a directory with officer photos (`public/images/officers`). Either move everything there or move everything into your new directory.

File: src/app/components/Projects/ApplyButton.tsx
Comment (0xfr4nkie) [inline src/app/components/Projects/ApplyButton.tsx:1]: Use camelCase as the naming convension for this file.

File: src/app/components/Projects/GitMonitor.tsx
Comment (0xfr4nkie) [inline src/app/components/Projects/GitMonitor.tsx:1]: Use camelCase as the naming convension for this file.

File: src/app/components/ui/ContentWrappers.tsx
Comment (0xfr4nkie) [inline src/app/components/ui/ContentWrappers.tsx:1]: Use camelCase as the naming convension for this file.

**Review by 0xfr4nkie — CHANGES_REQUESTED** [review]
Just a few minor changes. Everything else looks great.

File: public/assets/people/Frank_Lin.jpg
Comment (PitaFLAME) [reply public/assets/people/Frank_Lin.jpg:1]: That's the plan- haven't done it yet.  Not necessary for first day.

File: yarn.lock
Comment (PitaFLAME) [reply yarn.lock:1]: removed.

File: package.json
Comment (PitaFLAME) [reply package.json:1]: added.

File: public/assets/people/Frank_Lin.jpg
Comment (PitaFLAME) [reply public/assets/people/Frank_Lin.jpg:1]: For reference, I'm leaving some of this for contributors.

File: src/app/components/Projects/ApplyButton.tsx
Comment (PitaFLAME) [reply src/app/components/Projects/ApplyButton.tsx:1]: Different naming conventions for different types of files.  Components are in Pascal, utils are in lisp.

File: src/app/components/Projects/GitMonitor.tsx
Comment (PitaFLAME) [reply src/app/components/Projects/GitMonitor.tsx:1]: Different naming conventions for different types of files.  Components are in Pascal, utils are in lisp.

File: src/app/components/ui/ContentWrappers.tsx
Comment (PitaFLAME) [reply src/app/components/ui/ContentWrappers.tsx:1]: Different naming conventions for different types of files.  Components are in Pascal, utils are in lisp.

Comment (PitaFLAME) [reply src/app/components/ui/ContentWrappers.tsx:1]: Plus, many of the old components had different naming conventions.  Some of them are in Camel, some of them are in lisp.  I'm going to be turning All components into PascalCase.

**Comment by PitaFLAME** [comment]
god damn it frankie what did you do to me

**Comment by PitaFLAME** [comment]
I fixed it.  Don't ever ask me to change package managers again.

**Comment by PitaFLAME** [comment]
Give me the LGTM

**Review by 0xfr4nkie — APPROVED** [review]
LGTM 🚀

**Pull request merged**